src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtil.java

```diff
@@ -579,7 +579,7 @@ public static boolean isIdentifier(String str) {
      * @return true when the given string contains valid name.
      */
     public static boolean isName(String str) {
-        boolean isName = !str.isEmpty();
+        boolean isName = true;
```

Comment:
Why not `false`? Suppose the String is empty, is this really a name?

Reply:
Should be false.

Reply:
Sorry I have not visited this 
@nrmancuso https://github.com/checkstyle/checkstyle/pull/13475#discussion_r1278353998
https://github.com/checkstyle/checkstyle/blob/5b0a4425aedb83beea9bbc6301619492dabf7661/src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtil.java#L560-L590
The whole logic is written like this 
And for empty strings the test is already this 
https://github.com/checkstyle/checkstyle/blob/5b0a4425aedb83beea9bbc6301619492dabf7661/src/test/java/com/puppycrawl/tools/checkstyle/utils/CommonUtilTest.java#L392-L397

Reply:
https://github.com/checkstyle/checkstyle/pull/13475#discussion_r1278470409

I will change the logic soon

Reply:
Done